ci/scripts/go-executable-build.sh

```
@@ -0,0 +1,33 @@
+
+package=$1
+output_path=$(pwd)/$2
+app_dir=$4
+
+if [[ -z "$package" ]]; then
+  echo "usage: $0 <package-name>"
+  exit 1
+fi
+package_name=$package
+echo "cleaning dist..."
+rm -r $output_path
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)